### PR TITLE
Ana

### PR DESCRIPTION
**Pull request by AnastasiaPopiolek**

### CODE REVIEW

File: KITE-Janus-Test/src/main/java/io/cosmosoftware/kite/janus/JanusStreamingTest.java [hunk 1]
```diff
@@ -17,9 +18,13 @@ protected void populateTestSteps(TestRunner runner) {
     try {
       WebDriver webDriver = runner.getWebDriver();
       runner.addStep(new StartDemoStep(webDriver, this.url));
+      runner.addStep(new JoinStreamingStep(webDriver, "videoLive"));
       runner.addStep(new FirstVideoCheck(webDriver));
-      runner.addStep(new ReceiverVideoCheck(webDriver));
+      //see how the audioCheck can be adapted for the check of a live audio streaming
+      runner.addStep(new JoinStreamingStep(webDriver, "audioLive"));
+//      runner.addStep(new AudioCheck(webDriver));
```
Comment (ManuCosmo) [inline KITE-Janus-Test/src/main/java/io/cosmosoftware/kite/janus/JanusStreamingTest.java]: delete commented out code

File: KITE-Janus-Test/src/main/java/io/cosmosoftware/kite/janus/checks/FirstVideoCheck.java [hunk 2]
```diff
@@ -31,16 +31,22 @@ public String stepDescription() {
   protected void step() throws KiteTestException {
     try {
       final JanusPage janusPage = new JanusPage(this.webDriver, logger);
-      janusPage.waitForLocalStreamHeaderVisibility(5);
+//      janusPage.waitForLocalStreamHeaderVisibility(5);
```
Comment (ManuCosmo) [inline KITE-Janus-Test/src/main/java/io/cosmosoftware/kite/janus/checks/FirstVideoCheck.java]: delete commented out code

File: KITE-Janus-Test/src/main/java/io/cosmosoftware/kite/janus/pages/JanusPage.java [hunk 3]
```diff
@@ -41,6 +41,12 @@
   @FindBy(id="curres")
   private WebElement currentResolutionPrint;
 
+  /**
+   * Echo Test
+   */
+
+  //when simulcast=true
```
Comment (ManuCosmo) [inline KITE-Janus-Test/src/main/java/io/cosmosoftware/kite/janus/pages/JanusPage.java]: does simulcast=true work  on all Janus demos?

Comment (AnastasiaPopiolek) [reply KITE-Janus-Test/src/main/java/io/cosmosoftware/kite/janus/pages/JanusPage.java]: According to janus webRTC website, simulcast=true should work on:
-Echo Test
-Video Call Test
-Video Room Test
But I could not make it work on video call test (no button appears on the callee side)

**Pull request merged**